### PR TITLE
raspberry-pi-imager: Add version 1.5

### PR DESCRIPTION
**Pull request by alexchapar**
Tool for writing an Raspberry Pi OS images to SD cards

### CODE REVIEW

File: bucket/rpi-imager.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "version": "1.5",
+    "description": "Tool for writing an Raspberry Pi OS images to SD cards.",
+    "homepage": "https://github.com/raspberrypi/rpi-imager",
+    "license": {
+        "identifier": "Apache-2.0,GPL-2.1-or-later,LGPL-2.1-or-later,MIT,...",
+        "url": "https://github.com/raspberrypi/rpi-imager/blob/qml/license.txt"
+    },
```
Comment (Ash258) [inline bucket/rpi-imager.json:8]:
```suggestion
    "license": "Apache-2.0",
```

image

File: bucket/rpi-imager.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "url": "https://github.com/raspberrypi/rpi-imager/releases/download/v1.5/imager-1.5.exe#/dl.7z",
+    "hash": "sha256:9c62170a4180d64dfffc6f2af4bc5f3dc04cafd99c2511ed3af38dad9e2173e2",
+    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninst*\" -Force -Recurse",
+    "bin": "rpi-imager.exe",
```
Comment (Ash258) [inline bucket/rpi-imager.json:12]:
```suggestion
```

File: bucket/rpi-imager.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+    "version": "1.5",
+    "description": "Tool for writing an Raspberry Pi OS images to SD cards.",
+    "homepage": "https://github.com/raspberrypi/rpi-imager",
```
Comment (Ash258) [inline bucket/rpi-imager.json:4]:
```suggestion
    "homepage": "https://www.raspberrypi.org/software",
```

File: bucket/rpi-imager.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    "bin": "rpi-imager.exe",
+    "shortcuts": [
+        [
+            "rpi-imager.exe",
+            "Raspberry PI Imager"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/raspberrypi/rpi-imager"
+    },
```
Comment (Ash258) [inline bucket/rpi-imager.json:21]:
```suggestion
    "checkver": "imager_([\\d.]+)\\.exe",
```

File: bucket/rpi-imager.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    "autoupdate": {
+        "url": "https://github.com/raspberrypi/rpi-imager/releases/download/v$version/imager-$version.exe"
```
Comment (Ash258) [inline bucket/rpi-imager.json:23]:
```suggestion
        "url": "https://github.com/raspberrypi/rpi-imager/releases/download/v$version/imager-$version.exe#/dl.7z"
```

File: bucket/rpi-imager.json [hunk 6]
```diff
@@ -0,0 +1,25 @@
+{
+    "version": "1.5",
+    "description": "Tool for writing an Raspberry Pi OS images to SD cards.",
+    "homepage": "https://github.com/raspberrypi/rpi-imager",
+    "license": {
+        "identifier": "Apache-2.0,GPL-2.1-or-later,LGPL-2.1-or-later,MIT,...",
+        "url": "https://github.com/raspberrypi/rpi-imager/blob/qml/license.txt"
+    },
+    "url": "https://github.com/raspberrypi/rpi-imager/releases/download/v1.5/imager-1.5.exe#/dl.7z",
+    "hash": "sha256:9c62170a4180d64dfffc6f2af4bc5f3dc04cafd99c2511ed3af38dad9e2173e2",
```
Comment (Ash258) [inline bucket/rpi-imager.json:10]:
```suggestion
    "hash": "9c62170a4180d64dfffc6f2af4bc5f3dc04cafd99c2511ed3af38dad9e2173e2",
```

File: bucket/rpi-imager.json [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninst*\" -Force -Recurse",
```
Comment (Ash258) [inline bucket/rpi-imager.json:11]:
```suggestion
    "pre_install": "Remove-Item \"$dir\\`$*\", \"$dir\\uninst*\" -Recurse",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]
`git mv bucket/rpi-imager.json bucket/raspbery-pi-imager.json`

**Comment by alexchapar** [comment]
Thanks for your comments. I applied them and rename the file as proposed.

**Review by Ash258 — APPROVED** [review]

**Pull request merged**